pandas/core/ops/__init__.py

```diff
@@ -660,14 +660,6 @@ def wrapper(self, other):
 
         res_name = get_op_result_name(self, other)
 
-        # TODO: shouldn't we be applying finalize whenever
-        #  not isinstance(other, ABCSeries)?
-        finalizer = (
-            lambda x: x.__finalize__(self)
-            if isinstance(other, (np.ndarray, ABCIndexClass))
-            else x
-        )
```

Comment:
Now it will always be calling finalize. So was the comment there wrong? What if other is a Series? Does this have an impact on the result?

Reply:
> Does this have an impact on the result?

In the overwhemingly-most-common case where there is no user-defined _metadata, no, this doesn't affect the result.  Otherwise, it affects the result when other is a Series.